components/relay/src/lib.rs

```diff
@@ -0,0 +1,350 @@
+mod error;
```

Comment:
All the .rs files should have licenses (once in m-c their linters will complain)

Reply:
Updated to include MPL header as in the other files. Hopefully all checks stay green!